src/main/java/com/jmb/springfactory/model/factory/user/UserSamples.java

```diff
@@ -15,25 +15,31 @@ private UserSamples() {}
   public static final String SURNAME_USER_TEST_1 = "Martin Perea";
   public static final String PHONENUMBER_USER_TEST_1 = "123456789";
   public static final String EMAIL_USER_TEST_1 = "juan@gmail.com";
-  public static final Rol ROL_USER_TEST_1 = RolFactory.createRol(RolSamples.ID_ROL_TEST_1, RolSamples.NAME_ROL_TEST_1);
+  public static final Rol ROL_USER_TEST_1 = RolFactory.createRol(RolSamples.ID_ROL_TEST_1, RolSamples.NAME_ROL_TEST_1, 
+      RolFactory.createPermissions());
   public static final RolDto ROLDTO_USER_TEST_1 = RolDtoFactory.createRol(RolSamples.ID_ROL_TEST_1,
       RolSamples.NAME_ROL_TEST_1);
+  public static final String PASSWORD_USER_1 = "PASSWORD_USER_1";
 
   public static final String NIF_USER_TEST_2 = "99999999R";
   public static final String NAME_USER_TEST_2 = "Paco";
   public static final String SURNAME_USER_TEST_2 = "Perez De La Rosa";
   public static final String PHONENUMBER_USER_TEST_2 = "999555666";
   public static final String EMAIL_USER_TEST_2 = "paco@gmail.com";
-  public static final Rol ROL_USER_TEST_2 = RolFactory.createRol(RolSamples.ID_ROL_TEST_2, RolSamples.NAME_ROL_TEST_2);
+  public static final Rol ROL_USER_TEST_2 = RolFactory.createRol(RolSamples.ID_ROL_TEST_2, RolSamples.NAME_ROL_TEST_2, 
+      RolFactory.createPermissions());
   public static final RolDto ROLDTO_USER_TEST_2 = RolDtoFactory.createRol(RolSamples.ID_ROL_TEST_2,
       RolSamples.NAME_ROL_TEST_2);
+  public static final String PASSWORD_USER_2 = "PASSWORD_USER_2";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PASSWORD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)